### PR TITLE
Turn on Lein :pedantic? and fix warnings

### LINKED ISSUE #65: Turn on Lein :pendantic? mode and fix errors

### PR DESCRIPTION
**Pull request by frenchy64**
Close #65 

New dependency conflicts/overrides:
[dependencies.pdf](https://github.com/threatgrid/clj-momo/files/4154690/dependencies.pdf)

Iroh passes with this branch https://github.com/threatgrid/iroh/pull/3143
Ctia passes with this branch https://github.com/threatgrid/ctia/pull/871

### CODE REVIEW

**Review by gbuisson — APPROVED** [review]
LGTM

**Review by msprunck — APPROVED** [review]
LGTM!

File: project.clj [hunk 1]
```diff
@@ -1,46 +1,50 @@
...
-                  ;; Protobuf-java is brought in by ClojureScript
-                  :exclusions [com.google.protobuf/protobuf-java]]]
+                 [riemann-clojure-client "0.5.1"]
+                 [metrics-clojure ~metrics-clojure-version]
+                 [metrics-clojure-jvm ~metrics-clojure-version]
+                 [metrics-clojure-ring ~metrics-clojure-version]
+                 [metrics-clojure-riemann ~metrics-clojure-version
+                  :exclusions [;renamed to (and incompatible with) `io.riemann/riemann-java-client`,
+                               ;which is provided by `riemann-clojure-client`.
+                               com.aphyr/riemann-java-client]]
```
Comment (msprunck) [inline project.clj]: Is it possible to use `[threatgrid/metrics-clojure-riemann "2.10.1"]` like you did in the IROH repo?

Comment (frenchy64) [reply project.clj]: Ugh, yes great point.

Comment (frenchy64) [reply project.clj]: Updated to `[threatgrid/metrics-clojure-riemann "2.10.1"]`.

**Pull request merged**